src/writing-your-own-application/generate-project/esp-template.md

```diff
@@ -1,7 +1,7 @@
 # Understanding esp-template
 
 Now that we know how to [generate a no_std project][generate-no-std], let's inspect what the generated
-project contains, try to understand every part and run it.
+project contains, try to understand every part and, run it.
```

Comment:
The serial comma (the Oxford comma) should be added before and. Suggest updating:

... try to understand every part of it, and run it.